explore/src/main/scala/explore/config/ConfigurationPanel.scala

```diff
@@ -93,12 +105,40 @@ object ConfigurationPanel {
       op.runS(SpectroscopyRequirementsData()).value
     }
 
+  // Unsafe due to the angle truncation but usabe for the UI
+  private val unsafePosAngleAngleLens: Lens[PosAngle, TruncatedPA] =
```

Comment:
Can this be a `SplitEpi`?

Reply:
Can you pass a `SplitEpi` to `View`?

Reply:
There is a `zoomSplitEpi` method in `lucuma.ui.implicits`

Reply:
This maybe going beyond what a `SplitEpi` can do. `SplitEpi` would require a function `Angle => PosAngle` and there isn't enough information to build that info as `PosAngle` is an ADT

Reply:
Oh ok. I was trying to avoid the unsafe lens and thinking we can provide an update function for the `Angle`. This would require turning things around, with a safe lens into the Angle and then a SplitEpi Angle <-> TruncatedAngle, but then we would be stuck with a TruncatedAngle in the model, which we probably don't want.

Maybe we can just zoom using a get and mod function instead of a lens?